pkg/etc/wavefront/wavefront-proxy/wavefront.conf.default

```diff
@@ -91,71 +68,136 @@ customSourceTags=fqdn, hostname
+#authMethod=NONE
+## URL for the token introspection endpoint used to validate tokens for incoming HTTP requests.
+## Required when authMethod is OAUTH2 or HTTP_GET ")
+#authTokenIntrospectionServiceUrl=https://auth.acme.corp/api/token/{{token}}/validate
+## Optional credentials for use with the token introspection endpoint if it requires authentication.
+#authTokenIntrospectionAuthorizationHeader=Authorization: Bearer <token>
+## Cache TTL (in seconds) for token validation results (re-authenticate when expired). Default: 600 seconds
+#authResponseRefreshInterval=600
+## Maximum allowed cache TTL (in seconds) for token validation results when token introspection service is
+## unavailable. Default: 86400 seconds (1 day)
+#authResponseMaxTtl=86400
+## Static token that is considered valid for all incoming HTTP requests. Required when authMethod = STATIC_TOKEN.
+#authStaticToken=token1234abcd
+
+############################################# LOGS TO METRICS SETTINGS #################################################
+## Port on which to listen for FileBeat data (Lumberjack protocol). Default: none
+#filebeatPort=5044
+## Port on which to listen for raw logs data (TCP and HTTP). Default: none
+#rawLogsPort=5045
+## Maximum line length for received raw logs (Default: 4096)
+#rawLogsMaxReceivedLength=4096
+## Maximum allowed request size (in bytes) for incoming HTTP requests with raw logs (Default: 16MB)
+#rawLogsHttpBufferSize=16777216
+## Location of the logsingestion.yaml configuration file
+#logsIngestionConfigFile=/etc/wavefront/wavefront-proxy/logsingestion.yaml
+
+########################################### DISTRIBUTED TRACING SETTINGS ###############################################
 ## Comma-separated list of ports to listen on for Wavefront trace data. Defaults to none.
 #traceListenerPorts=30000
+## Maximum line length for received spans and span logs (Default: 1MB)"
+#traceListenerMaxReceivedLength=1048576
+## Maximum allowed request size (in bytes) for incoming HTTP requests on tracing ports (Default: 16MB)")
+#traceListenerHttpBufferSize=16777216
+
 ## Comma-separated list of ports on which to listen on for Jaeger Thrift formatted data. Defaults to none.
-#traceJaegerListenerPorts=30001
+#traceJaegerListenerPorts=14267
 ## Custom application name for traces received on Jaeger's traceJaegerListenerPorts.
 #traceJaegerApplicationName=Jaeger
 ## Comma-separated list of ports on which to listen on for zipkin trace data over HTTP. Defaults to none.
-## Recommended value is 9411, which is the port Zipkin's server listens on and is the default
-configuration in Istio.
+## Recommended value is 9411, which is the port Zipkin's server listens on and is the default configuration in Istio.
 #traceZipkinListenerPorts=9411
 ## Custom application name for traces received on Zipkin's traceZipkinListenerPorts.
 #traceZipkinApplicationName=Zipkin
+## A comma separated, list of additional custom tag keys to include along as metric tags for the
```

Comment:
A comma separated, list of additional custom tag keys
=> Comma-separated list of additional custom tag keys

(no comma in this sentence. other changes nice to have.